index.Dockerfile389314389

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/operator-framework/upstream-opm-builder
```

Comment:
Should we check this file in and the index.db?

Reply:
I removed this... it was a failure on the index creation